src/object/expo_push_receipt_id.rs

```diff
@@ -1,12 +1,88 @@
-use serde::{Deserialize, Serialize};
+#[derive(
+    Clone, Debug, Eq, Hash, Ord, PartialEq, PartialOrd, serde::Deserialize, serde::Serialize,
+)]
+pub struct ExpoPushReceiptId(String);
 
-#[derive(Debug, Deserialize, Serialize, PartialEq, Clone)]
-pub struct ExpoPushReceiptId {
-    ids: Vec<String>,
+impl std::convert::From<ExpoPushReceiptId> for String {
+    fn from(expo_push_receipt_id: ExpoPushReceiptId) -> Self {
+        expo_push_receipt_id.0
+    }
+}
+
+impl std::convert::TryFrom<String> for ExpoPushReceiptId {
+    type Error = crate::CustomError;
```

Comment:
```suggestion
    type Err = crate::CustomError;
```

Reply:
The associated type name for TryFrom is Error.
https://doc.rust-lang.org/std/convert/trait.TryFrom.html